lib/perl/Genome/Config/AnalysisProject/SubjectMapping/Command/Predict/SomaticValidation.pm

```diff
@@ -0,0 +1,163 @@
+package Genome::Config::AnalysisProject::SubjectMapping::Command::Predict::SomaticValidation;
+
+use strict;
+use warnings;
+
+use Genome;
+
+class Genome::Config::AnalysisProject::SubjectMapping::Command::Predict::SomaticValidation {
+    is => 'Genome::Config::AnalysisProject::Command::Base',
+    has_input => [
+        file_path => {
+            is => 'Text',
+            shell_args_position => 2,
+            doc => 'output path for a newline-delimited, tab-separated list of samples and required whitespace'
+        }
+    ],
+    has_optional => [
+        normal_sample_common_names => {
+            is      => 'Text',
+            default => Genome::Model::ClinSeq::Command::UpdateAnalysis->__meta__->property('normal_sample_common_names')->default_value,
+            doc     => Genome::Model::ClinSeq::Command::UpdateAnalysis->__meta__->property('normal_sample_common_names')->doc,
+        },
+        tumor_sample_common_names => {
+            is      => 'Text',
+            default => Genome::Model::ClinSeq::Command::UpdateAnalysis->__meta__->property('tumor_sample_common_names')->default_value,
+            doc => Genome::Model::ClinSeq::Command::UpdateAnalysis->__meta__->property('tumor_sample_common_names')->doc,
+        },
+        sample_identifier => {
+            is => 'Text',
+            doc => 'The method used to resolve the identifier of the sample.',
+            default => 'name',
+            valid_values => ['name','id'],
+        },
+    ],
+};
+
+sub help_brief {
+    return 'Predict sample pairings in bulk for an AnalysisProject';
```

Comment:
It might be nice if this help contains a mention of `SomaticValidation` (to distinguish it from the `doc` for the command one level up the tree).